lib/screens/filters_screen.dart

```diff
@@ -1,18 +1,96 @@
 import 'package:flutter/material.dart';
 import 'package:meals_app/widgets/main_drawer.dart';
 
-class FiltersScreen extends StatelessWidget {
+class FiltersScreen extends StatefulWidget {
   static const routeName = '/filters';
 
+  @override
+  _FiltersScreenState createState() => _FiltersScreenState();
+}
+
+class _FiltersScreenState extends State<FiltersScreen> {
+  var _glutenFree = false;
+  var _vegetarian = false;
+  var _vegan = false;
+  var _lactoseFree = false;
+
+  Widget _buildSwitchListTile(
+    String title,
+    String description,
+    var currentValue,
+    Function updateValue,
+  ) {
+    return SwitchListTile(
+      title: Text(title),
+      value: currentValue,
+      subtitle: Text(description),
+      onChanged: updateValue,
+    );
+  }
+
+  void changeGlutenFree(newValue) {
+    setState(() => _glutenFree = newValue);
+  }
+
+  void changeLactoseFree(newValue) {
+    setState(() => _lactoseFree = newValue);
+  }
+
+  void changeVegetarian(newValue) {
+    setState(() => _vegetarian = newValue);
+  }
```

Comment:
please apply what needs to be done based on [this comment](https://github.com/MapleSyyrup/Meals_App/pull/15/files#r546799208)